connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -285,23 +297,26 @@ public Response restartConnector(final @PathParam("connector") String connector,
 
     @PUT
     @Path("/{connector}/pause")
+    @Operation(summary = "Pause the specified connector")
```

Comment:
Mention the semantics of pausing an already paused connector.

Reply:
I think nothing happens when pausing an already paused connector. Isn't it the case?